media/cat-logo.svg

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
The same file can be found at `docs/_static/img/cat-logo.svg`.

Reply:
Maybe this should live in media/ only, although then in docs/config.py there is a reference that goes ../../  which is a bit ugly

Reply:
If you really wanna keep both file references, maybe symlinks can be used here. Windows machine needs extra configuration for it to work though for git.

Reply:
removed

Reply:
Oh cool. The packaged `medcat-*.tar.gz` will definitely be smaller.